### PR TITLE
Update spdlog 1.3.1 to v1.5.0

### PR DESCRIPTION
**Pull request by Ninic0c0**
Please see https://github.com/gabime/spdlog/releases/tag/v1.5.0

### CODE REVIEW

File: meta-oe/recipes-support/spdlog/spdlog_1.5.0.bb [hunk 1]
```diff
@@ -3,8 +3,7 @@ HOMEPAGE = "https://github.com/gabime/spdlog/wiki"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
 
-SRCREV = "a7148b718ea2fabb8387cb90aee9bf448da63e65"
-SRC_URI = "git://github.com/gabime/spdlog.git;protocol=git;branch=v1.x;"
+SRC_URI = "git://github.com/gabime/spdlog.git;protocol=git;branch=v1.x;tag=v1.5.0;"
```
Comment (kraj) [inline meta-oe/recipes-support/spdlog/spdlog_1.5.0.bb]: Let us use SRCREV corresponding to v1.5.0, using tags still forces bitbake to ping the upstream git and breaks  builds where BB_NO_NETWORK is used.

File: meta-oe/recipes-support/spdlog/spdlog_1.5.0.bb [hunk 2]
```diff
@@ -17,3 +16,4 @@ inherit cmake
 # Header-only library
 RDEPENDS_${PN}-dev = ""
 RRECOMMENDS_${PN}-dbg = "${PN}-dev (= ${EXTENDPKGV})"
+
```
Comment (kraj) [inline meta-oe/recipes-support/spdlog/spdlog_1.5.0.bb]: drop this empty line

File: meta-oe/recipes-support/spdlog/spdlog_1.5.0.bb (hunk 1 above)
Comment (Ninic0c0) [reply meta-oe/recipes-support/spdlog/spdlog_1.5.0.bb]: Good to know :)

File: meta-oe/recipes-support/spdlog/spdlog_1.5.0.bb (hunk 2 above)
Comment (Ninic0c0) [reply meta-oe/recipes-support/spdlog/spdlog_1.5.0.bb]: done.

**Comment by kraj** [comment]
please squash it into single commit.

**Comment by Ninic0c0** [comment]
Let me apologize but I'm not enough competent to do that properly.
You can close/delete this PR.

**Comment by kraj** [comment]
I have squashed it this time and staged it in master-next for testing

**Comment by kraj** [comment]
476071913bcc18147f39fec8bc613967a4c2fb63 appliied

**Pull request closed without merging**